.github/workflows/ci.yml

```diff
@@ -66,21 +66,29 @@ jobs:
             clang: git build-essential pkg-config python3 curl openjdk-11-jdk pkg-config libncurses-dev libxml2-utils libxml2-dev g++-14=14.2.0-4ubuntu2~24.04
             msvc: ''
           extra-values: |
-            llvm-hash: a1b6e7ff393533a5c4f3bdfd4efe5da106e2de2b
-            llvm-build-preset-prefix: {{#if optimized-debug}}debwithopt{{else}}{{{lowercase build-type}}}{{/if}}
+            use-libcxx: {{#if (and (ieq compiler 'clang') (or (eq major 19) (eq major 20) (eq major 21))) }}true{{else}}false{{/if}}
```

Comment:
clang will be always tested with libcxx now? Even on linux? Even when unrelated to sanitizers.

Reply:
I didn't find an operator equivalent to `>=`, it should be spelled 'ge' I think for consistency with naming of existing operators.

Reply:
Yeah, I felt that there was some value in using these on clang even without sanitizers, because at least the libcxx built here will be controlled by the same flags, so it will have debug info, will have runtime assertions if we choose to enable it, and so on. It would give some extra variation on top of what we already do on the GCC jobs.